upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -1196,13 +1196,29 @@ func awsBuildCloudInstanceGroup(ctx context.Context, c AWSCloud, cluster *kops.C
 		return nil, fmt.Errorf("failed to fetch instances: %v", err)
 	}
 
+	scalingReq := &autoscaling.DescribeScalingActivitiesInput{
```

Comment:
I don't know whether this gets expensive, but I would if we should plumb through a flag here as to whether we want this information (or add a method or func to CloudInstanceGroup that could query it on-demand)